client/me/purchases/cancel-private-registration/index.jsx

```diff
@@ -20,7 +20,8 @@ import SimpleNotice from 'notices/simple-notice';
 const CancelPrivateRegistration = React.createClass( {
 	getInitialState() {
 		return {
-			disabled: false
+			disabled: false,
+			cancelling: false
```

Comment:
Do we really need to introduce another attribute? `disabled` and `cancelling` handle the same thing apparently. I could see a reason to make the distinction if there was a form with data loading from the API but here we only need to prevent users from submitting the form when they click the `Cancel Private Registration` button.